engines/dfc_provider/app/controllers/dfc_provider/api/products_controller.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+# Controller used to provide the API products for the DFC application
+module DfcProvider
+  module Api
+    class ProductsController < ::ActionController::Metal
+      include ActionController::Head
+      include AbstractController::Rendering
+      include ActionController::Rendering
+      include ActionController::Renderers::All
+      include ActionController::MimeResponds
+      include ActionController::ImplicitRender
+      include AbstractController::Callbacks
```

Comment:
Ah right, actually I started with less and add more and more code slowly :) I will update this.

Reply:
Ok not working because of this:
´´´
include CanCan::ControllerAdditions
include Spree::Core::ControllerHelpers::Auth
´´´
It is imposing an authentication layer which is not used on this API.